tests/demo_test.py

```diff
@@ -1,20 +1,63 @@
-import requests
 from utilities.configuration import *
 from data.payloads import *
-from data.payload_data import *
+from data.user import *
```

Comment:
warto wiedzieć, że zachęca się do używania dokładnych importów:
https://stackoverflow.com/questions/3615125/should-wildcard-import-be-avoided